tests/src/Kernel/PriceTwigExtensionTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\Tests\commerce\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+
+/**
+ * Tests the price twig filter.
+ *
+ * @group commerce
+ */
+class PriceTwigExtensionTest extends KernelTestBase {
+
+  /**
+   * Load in USD currency.
+   */
+  public function setUp() {
+    parent::setUp();
+    $this->container->get('commerce_price.currency_importer')->import('USD');
+  }
+
+  public static $modules = [
+    'system',
+    'commerce',
+    'commerce_price',
+    'commerce_test',
+  ];
+
+  /**
+   * Tests an improperly formatted price array.
+   */
+  public function testBrokenPrice() {
+    $theme = ['#theme' => 'broken_commerce_price'];
+    try {
+      $this->render($theme);
+    }
+    catch (\InvalidArgumentException $e) {
```

Comment:
Don't do try / catch in tests. Instead, use $this->setExpectedException(\InvalidArgumentException::class); before the render call.